System.IO.Abstractions.TestingHelpers.Tests/MockFileCreateTests.cs

```diff
@@ -155,12 +155,12 @@ public void MockFile_Create_ShouldThrowDirectoryNotFoundExceptionIfCreatingAndPa
         {
             // Arrange
             var fileSystem = new MockFileSystem();
+            var file = XFS.Path("C:\\path\\NotFound.ext");
```

Comment:
This test was not normalizing the path which caused test failures on Unix, so I fixed this.